style.scss

```
@@ -0,0 +1,346 @@
+$cBrown: #bfb2ab;
+
+* {
+    box-sizing: border-box;
+    font-family: 'Oswald', sans-serif;
+    font-family: 'Rasa', serif;
+    font-family: 'Spectral', serif;
+    margin: 0;
+    padding: 0;
+}
+
+
+
+// ========================= mobile ============================================
+@media screen and (max-width:1030px) {
+// ========================= top page ====================
+body {
+    position: relative;
+}
+
+.page1 {
+    height: 100vh;
+
+    .oneArea {
+        height: 100vh;
+        position: absolute;
+        top: 0px;
+        right: 0px;
+        border-bottom: 100vh solid $cBrown;  /* 三角を白にする */ 
+        border-left: 100vw solid transparent;
+    }
+
+    .top {
+        
+        & img {
+            position: absolute;
+            top: 30vh;
+            left: 8px;
+            width: 200px;
+        }
+        &-text {
+
+            & h1 {
+                font-size: 20px;
+                width: 190px;
+                position: absolute;
+                top: 20vh;
+                left: 20px;
```

Just for my little suggestion, this is more looks better.
```suggestion
                left: 30px;
```

---

Fixed! Thank you.